src/Libraries/Microsoft.Extensions.AI.Abstractions/README.md

```diff
@@ -97,14 +97,12 @@ while (true)
     Console.Write("Q: ");
     history.Add(new(ChatRole.User, Console.ReadLine()));
 
-    List<ChatResponseUpdate> updates = [];
     await foreach (var update in client.GetStreamingResponseAsync(history))
     {
         Console.Write(update);
+        history.AddMessages(update);
```

Comment:
This isn't the right fix. The right fix is to keep exactly what was there before and add here as L103 `updates.Add(update);`.